layer/scripts/gen_all_ext_profile.py

```diff
@@ -0,0 +1,530 @@
+import os
+import xml.etree.ElementTree as etree
+import math
+import argparse
+from fractions import Fraction
+
+PROFILES = '''
+    "profiles": {
```

Comment:
My general comment would be that, instead of using strings, you should prefer constructing JSONs in python by simply using python dicts and lists and then dump the formatted JSON output using `json.dumps(data, indent=4)`.

You can check it out how that works in the `VulkanProfilesSchemaGenerator` of `genvp.py`. In particular, you can check the `gen_schema` method which contains the dict/list based skeleton of the generated JSON schema.